python/paddle/fluid/op.py

```diff
@@ -24,13 +26,13 @@ def get_all_op_protos():
     protostrs = core.get_all_op_protos()
     ret_values = []
     for pbstr in protostrs:
-        op_proto = framework_pb2.OpProto.FromString(str(pbstr))
+        op_proto = framework_pb2.OpProto.FromString(six.binary_type(pbstr))
         ret_values.append(op_proto)
     return ret_values
 
 
 def is_str(s):
-    return isinstance(s, str) or isinstance(s, unicode)
+    return isinstance(s, str) or isinstance(s, str)
```

Comment:
duplicated? ```is_str``` should be handled better here for py2 and py3?

Reply:
Yes, actually, ```isinstance(s, str) or isinstance(s, unicode)``` could be replaced with ```isinstance(v, six.string_types)```, because ```str``` and ```unicode``` in python2 are inherited from ```base_string```, i.e.. ```six.string_types```